extensions/amp-imgur/0.1/amp-imgur.js

```diff
@@ -94,21 +87,26 @@ export class AmpImgur extends AMP.BaseElement {
       encodeURIComponent(this.imgurid_) + '/embed?pub=true';
     this.applyFillContent(iframe);
     this.element.appendChild(iframe);
-    return this.iframePromise_ = this.loadPromise(iframe);
+    return this.loadPromise(iframe);
   }
 
   /** @private */
   hadleImgurMessages_(event) {
+    if (event.origin != 'https://imgur.com' ||
+        event.source != this.iframe_.contentWindow) {
+      return;
+    }
+    if (!event.data || !(isObject(event.data))) {
+      return;
+    }
     const data = isObject(event.data) ? event.data : tryParseJson(event.data);
-    if(data.message == 'resize_imgur') {
+    if (data.message == 'resize_imgur') {
       const height = data.height;
 
       this.getVsync().measure(() => {
```

Comment:
This vsync call is unnecessary now that we're not reading the `outerHeight`. We can just call `#attemptChangeHeight`